ci/scripts/run-ecs-db-migration.js

```diff
@@ -90,7 +99,13 @@ const run = async function run () {
     }
 
     console.log(`Succesfully scheduled db migration task: ${runResult.tasks[0].taskArn}`)
-    console.log('View progress of the migration at: https://gds.splunkcloud.com/en-GB/app/gds-004-pay/migration_status')
```

Comment:
I'd leave this log in at the start, if a migration takes a while the dev can go watch the logs in splunk at least to see what's happening

Reply:
Fixed